tests/xtimer_benchmarks/main.c

```diff
@@ -0,0 +1,576 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       RIOT timer benchmarks
+ *
+ * @author      M. Aiman Ismail <muhammadaimanbin.ismail@haw-hamburg.de>
+
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "periph/gpio.h"
+#include "test_utils/expect.h"
+#include "fmt.h"
+
+#include "shell.h"
+#include "test_helpers.h"
+#include "sc_args.h"
+#include "random.h"
+#include "timex.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+#include "log.h"
+
+#ifndef PARSER_DEV_NUM
+#define PARSER_DEV_NUM 0
+#endif
+
+#define TEST_REPEAT     (50)
```

Comment:
This will complicate the testing as PHILIP can only hold 128 values in its buffer. Making it configurable might cause others to use higher test repeat than PHILIP can handle. Maybe we can add checks for that but I think we can make it configurable later when there's a need.